recipes/r-here/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "here" %}
+{% set version = "0.1" %}
+
+package:
+  name: r-{{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/{{ name }}_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/{{ name }}/{{ name }}_{{ version }}.tar.gz
+  sha256: 5ba5e7d9002fc85bc633706b7ee486505c12b0bf5ecb177028bec6db706f65ed
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-rprojroot >=1.2
+
+  run:
+    - r-base
+    - r-rprojroot >=1.2
+
+test:
+  commands:
+    - $R -e "library('{{ name }}')"  # [not win]
+    - "\"%R%\" -e \"library('{{ name }}')\""  # [win]
+
+about:
+  home: http://krlmlr.github.io/here 
+  license: GPL-3
+  summary: Constructs paths to your project's files based on heuristics.
+  description: |
+    Constructs paths to your project's files. The 'here()' function uses a reasonable 
+    heuristics to find your project's files, based on the current working directory at the time 
+    when the package is loaded. Use it as a drop-in replacement for 'file.path()', it will always 
+    locate the files relative to your project root.
+  license_family: GPL
+  dev_url: https://github.com/krlmlr/here
```

Comment:
Please add the following license information here:
  license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'  # [unix]
  license_file: '{{ environ["PREFIX"] }}\R\share\licenses\GPL-3'  # [win]